backend/webui_service/webui_init.go

```diff
@@ -190,19 +190,22 @@ func (webui *WEBUI) Start() {
 	go func() {
 		httpAddr := ":" + strconv.Itoa(factory.WebUIConfig.Configuration.CfgPort)
 		logger.InitLog.Infoln("Webui HTTP addr", httpAddr)
-		if factory.WebUIConfig.Info.HttpVersion == 2 {
+		tlsConfig := factory.WebUIConfig.Configuration.TLS
+		if factory.WebUIConfig.Info.HttpVersion == 2 || tlsConfig != nil {
```

Comment:
Why do you need this `|| tlsConfig != nil`? My understanding is that we are interested in Http version 2.0, which can be either insecure (http => `ListenAndServe`) or secure (https => `ListenAndServeTLS`), and this is checked by `if tlsConfig != nil` in line 204. However, by having `|| tlsConfig != nil` here, we are saying that the Http version can be <= 2. Is this the intention of this change?
```suggestion
		if factory.WebUIConfig.Info.HttpVersion == 2 {
```

Reply:
Agree.